### PR TITLE
Image rotation support

### PR DESCRIPTION
**Pull request by ananyajana**
Rotate JPG and PNG images.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,7 +58,7 @@ For example, if an image has a resolution of 2048x1365 and is being resized to 1
       -r, --recursive    process directories recursively [default: off]
       -w, --overwrite    overwrite source images [default: off]
       -z, --debug        enable debug logs [default: off]
-
+      -o deg, --rotate deg  rotate the image by specified angle
```
Comment (jarun) [inline README.md]: Please keep it as the 4th option.

File: imgd [hunk 2]
```diff
@@ -305,12 +361,12 @@ def parse_args(args=None, namespace=None):
...
 def main():
     global HRES, VRES, scale, adapt, convert, dot, eraseexif, force, \
-           enlarge, optimize, quiet, recurse, overwrite, debug, no_res_opt
+           enlarge, optimize, quiet, recurse, overwrite, debug, no_res_opt, rotate
 
     args = parse_args()
 
-    if (args.res is not None and args.scale is not None):
-        print('specify either resolution or scale')
+    if (args.res is not None and args.scale is not None) or (args.res is not None and args.rotate is not None) or (args.rotate is not None and args.scale is not None):
```
Comment (jarun) [inline imgd]: Add a separate simpler check below this.

File: imgd [hunk 3]
```diff
@@ -65,11 +66,64 @@ def traverse_dir(path):
...
+
+        if overwrite:
+            if converted:
+                os.remove(src)
+                shutil.move(name + pad + '.jpg', name + '.jpg')
+            else:
+                shutil.move(name + pad + ext, src)
+    except OSError as e:
+        print(e)
+
```
Comment (jarun) [inline imgd]: Remove extra newlines.

File: README.md (hunk 1 above)
Comment (ananyajana) [inline README.md]: Done.

File: imgd (hunk 3 above)
Comment (ananyajana) [inline imgd]: Done.

**Comment by ananyajana** [comment]
Addressed review comments for image rotation functionality

**Comment by jarun** [comment]
Merged.

**Pull request merged**